databases/week1/sql_queries.sql

```diff
@@ -0,0 +1,89 @@
+--Find out how many tasks are in the task table
+SELECT
+  count(id)
+FROM
+  task;
+
+--Find out how many tasks in the task table do not have a valid due date
+SELECT
+  count(id)
+FROM
+  task
+WHERE
+  due_date IS NULL;
+
+--Find all the tasks that are marked as done
+SELECT
+  *
```

Comment:
Here i wouldn't select all columns, because the resulting table gets very wide. Instead i would just select the task title and the status name, that would be enough to see which tasks are done.

Reply:
It was not mentioned what to show in the table so I selected all the columns but now I have changed it to show the title and the status name.

Reply:
Yes, i know i didn't say which fields to select, it's more of a good practice.